cmd/webhook/main.go

```diff
@@ -143,19 +144,25 @@ func main() {
 		NodeFlags:            nodeFlags,
 		Namespace:            opt.namespace,
 		VersionConstraints:   constraint,
+
+		// we could change this to get the CertDir from the configured CertName
+		// and KeyName, but doing so does not bring us any benefits but would
+		// technically break compatibility.
```

Comment:
technically breaks compatibility if we enforce TLS cert and key being in the same directory